add rucio jlab vre documentation (#9)
vre-hub/vre-hub.github.io: docs/extensions/rucio-jupyterlab/index.md

# Rucio JuputerLab Extension

[![Test](https://github.com/rucio/jupyterlab-extension/actions/workflows/test.yml/badge.svg)](https://github.com/rucio/jupyterlab-extension/actions/workflows/test.yml)
[![Build and Publish Extension](https://github.com/rucio/jupyterlab-extension/actions/workflows/build-and-publish-tagged.yml/badge.svg)](https://github.com/rucio/jupyterlab-extension/actions/workflows/build-and-publish-tagged.yml)

:::tip[IMPORTANT]

> Please note that this documentation is specific for the deployment of the Rucio extension in the 

Reviewer: ```suggestion 
Please note that this documentation is specific to the deployment of the Rucio extension within the 
```
